modules/custom/az_gdpr_consent/az_gdpr_consent.install

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the az_gdpr_consent module.
+ */
+
+use Drupal\Core\Url;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Implements hook_install().
+ */
+function az_gdpr_consent_install() {
```

Comment:
Has using our Quickstart config provider (`config/quickstart`) already been discussed as an alternative to forcing the overrides to be installed here?